queries.sql

```diff
@@ -145,3 +145,55 @@ GROUP BY Species;
 SELECT Species, AVG(escape_attempts) FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY Species;
+
+-- Day 3
+-- Write queries (using JOIN) to answer the following questions:
+--   What animals belong to Melody Pond?
+SELECT animals.id AS id, name AS name, owner_id AS owner_id, full_name AS owner_name
+FROM animals
+INNER JOIN owners ON animals.owner_id = owners.id
+WHERE full_name = 'Melody Pond';
+-- DONE
+
+--   List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.id AS id, animals.name as animal_name, species_id AS specie_id, species.name AS specie_type
+FROM animals
+INNER JOIN species ON animals.species_id = species.id 
+WHERE species.name = 'Pokemon';
+-- DONE
+
+--   List all owners and their animals, remember to include those that don't own any animal.
+SELECT owners.id AS owner_id, full_name AS owner_name, animals.id AS animal_id, name as animal_name
+FROM animals
+RIGHT JOIN owners ON animals.owner_id = owners.id;
+-- DONE
+
+--   How many animals are there per species?
+SELECT species.name As specie, COUNT(species.name) AS count
+FROM animals
+INNER JOIN species ON animals.species_id = species.id
+GROUP BY species.name;
+-- DONE
+
+--   List all Digimon owned by Jennifer Orwell.
+SELECT animals.name, owners.full_name
+FROM animals
+INNER JOIN owners ON animals.owner_id = owners.id
+INNER JOIN species ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell';
+-- DONE
```

Comment:
- [ ] Great work on your query! 👏 However, I noticed a small issue that needs attention. When I examined the data you inserted into the database, it appears that `Jennifer Orwell` owns two animals, but only one of them is of type `Digimon`. 

   - Your query is designed to retrieve all `Digimons` owned by `Jennifer Orwell`, but it's currently returning both of her animals, including the one that isn't a `Digimon`. To ensure that your query provides accurate results, I recommend making the necessary adjustments to filter out only the `Digimons` owned by `Jennifer Orwell`. 

Please don't forget to update your pull request summary with the screenshot of the revised query results. 🔄 
Happy coding! 😊🚀🍀

Reply:
Made required changes

- List all Digimon owned by Jennifer Orwell.

![Screenshot 2023-09-30 130345](https://github.com/vikas924/Vet-clinic-database/assets/121718906/c0b37ac9-cb12-4473-aaed-22d5ca70b63a)